manifests/rawx.pp

```diff
@@ -19,7 +19,7 @@
   $typesConfig                = '/etc/mime.types',
   $grid_fsync                 = 'enabled',
   $grid_fsync_dir             = 'enabled',
-  $httpd_mpm                  = {'worker'=>{'StartServers'=>3,'MinSpareThreads'=>75,'MaxSpareThreads'=>250,'ThreadsPerChild'=>25,'MaxRequestsPerChild'=>0,'ServerLimit'=>16}},
+  $httpd_mpm                  = {'worker'=>{'StartServers'=>1,'MinSpareThreads'=>32,'MaxSpareThreads'=>256,'ThreadsPerChild'=>256,'MaxRequestsPerChild'=>0,'MaxRequestWorkers'=>4096,'ServerLimit'=>16}},
```

Comment:
Why hardcoding `MaxRequestWorkers` to 4096 ? As it is computed from ServerLimit by ThreadsPerChild by default ? (which is, in that cache allready 4096)